recipes/opencl-clang/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "opencl-clang" %}
+{% set version = "9.0.0" %}
+{% set llvm_version = version %}
+
+package:
+  name: intel-opencl-clang
+  version: {{ version }}
+
+source:
+  url: https://github.com/intel/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: f98b9122da4066e96853429a3cff9e6c0bfb0fee14fa2be5a22e43546a5cfd93
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - llvmdev   {{ llvm_version }}
+    - llvm      {{ llvm_version }}
+    - clangdev  {{ llvm_version }}
+    - llvm-spirv   {{ llvm_version }}*
```

Comment:
Is the `*` needed? This is done automatically by conda normally.

Reply:
Fixed